docs/docutils.conf

```diff
@@ -0,0 +1,2 @@
+[restructuredtext parser]
```

Comment:
Could we add a comment to this file with a short explanation for why this is needed and a link to the related Sphinx issue that you opened?